templates/CRM/Event/Form/Task/Delete.tpl

```diff
@@ -13,10 +13,9 @@
   {icon icon="fa-info-circle"}{/icon}
   <div>
     <p>{ts}Are you sure you want to delete the selected participations? This delete operation cannot be undone and will delete all transactions and activity associated with these participations.{/ts}</p>
-        <p>{include file="CRM/Event/Form/Task.tpl"}</p>
```

Comment:
This doesn't do anything?

Reply:
@colemanw not is `$rows`is not present

![image](https://github.com/civicrm/civicrm-core/assets/336308/e21540d4-fa91-4985-83a5-ff293446e25a)

Reply:
Actually there is one line to bring back - one moment caller

Reply:
@colemanw fixed per above